sample-lnd.conf

```diff
@@ -486,13 +486,12 @@ bitcoin.simnet=true
 ; signet network seed nodes
 ; bitcoin.signetseednode=123.45.67.89
 
-; Use the btcd back-end
+; Specify the chain back-end. Options are btcd, bitcoind and neutrino.
+;
+; NOTE: Please note that switching from a full back-end (btcd/bitcoind) to a
```

Comment:
I think it is 

> switching from a full back-end (btcd/bitcoind) to light back-end (neutrino) is not *officially* supported and switching from a light back-end to a full back-end is won't work

Am I missing something?

Reply:
I have switched both ways and my node still works. It just messed up my channel db and @guggero helped me out fixing it. It can work without problems I think. It's just not something people should do (eg: it's not supported)...

Reply:
> It just messed up my channel db and @guggero helped me out fixing it. It can work without problems I think.

Can you point out what was messed up and what was done to fix it?  This could be useful to properly add support.

Reply:
> It just messed up my channel db and @guggero helped me out fixing it. It can work without problems I think.

Can you say what was messed up, and what was done to fix it?  This could be useful to add support for switching.

Reply:
That's by now months ago... I had to drop a channel using chantools, because (meta?) data was not filled correctly iirc.